Examples/MediumViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MediumViewController.swift
+//  CollectionKit
+//
+//  Created by yansong li on 2017-09-03.
+//  Copyright © 2017 lkzhao. All rights reserved.
+//
+
+import CollectionKit
+import UIKit
+
+class MediumViewController: UIViewController {
```

Comment:
Maybe rename this to "ArticleExampleVC"?

Reply:
Nice point